src/v0/destinations/google_adwords_enhanced_conversions/transform.js

```diff
@@ -47,15 +47,21 @@ const responseBuilder = async (metadata, message, { Config }, payload) => {
   }
   const filteredCustomerId = removeHyphens(customerId);
 
-  response.endpoint = `${BASE_ENDPOINT}/${filteredCustomerId}:uploadConversionAdjustments`;
   response.body.JSON = payload;
   const accessToken = getAccessToken(metadata, 'access_token');
   response.headers = {
     Authorization: `Bearer ${accessToken}`,
     'Content-Type': JSON_MIME_TYPE,
     'developer-token': getValueFromMessage(metadata, 'secret.developer_token'),
   };
-  response.params = { event, customerId: filteredCustomerId };
+  const filteredLoginCustomerId = removeHyphens(loginCustomerId);
```

Comment:
duplicate line

Reply:
This is not a duplicate. This is for loginCustomerId and another is for customerId.